examples/remote_execution/buildbarn/platforms/defs.bzl

```diff
@@ -21,6 +21,8 @@ def _platforms(ctx):
             # Set those up based on what workers you've registered with Build Barn.
             remote_execution_properties = {
                 "OSFamily": "Linux",
+                # This could be any Docker image.
```

Comment:
I don't think it can actually be any Docker image? It's got to be the one you did start your Build Barn cluster with, no?

Reply:
No, you are right, and your base example did not have the comment. I just copy-pasted it from the Engflow example.

I'll remove it.